### PR TITLE
William Loy add exchange mailboxes

### PR DESCRIPTION
**Pull request by DonNateR**
From PR #2084

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-00560b1529/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3e389b7d78/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7965d50229/how-to/).

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. Given how similar this process is to the one for adding Rackspace Email mailboxes, I edited this article to be basically the same as the other one, so all the edits should look familiar! :) Please let me know if you have any questions. Thanks!

[Add Microsoft Exchange mailboxes_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1069683/Add.Microsoft.Exchange.mailboxes_editsKH.pdf)
[Add Microsoft Exchange mailboxes_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1069684/Add.Microsoft.Exchange.mailboxes_editsKH.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b1bfbec8f1/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a67c4badbc/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c296e2cf9d/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-da2b3abe83/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-86f274c694/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-86b7753410/how-to/)

File: content/exchange/add-microsoft-exchange-mailboxes.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+### Add a single Exchange mailbox
+
+1. Log in to the [Cloud Office Control Panel](https://cp.rackspace.com) using your Rackspace Cloud Office admin ID and password.
+
+2. In the Microsoft Exchange section, click **Mailboxes**.
+
+   <img src="{% asset_path exchange/add-microsoft-exchange-mailboxes/add-hex-mailbox-sc1.png %}" />
+
+3. If you have multiple domains, select the domain to which you want to add a mailbox to.
```
Comment (stephamon) [inline content/exchange/add-microsoft-exchange-mailboxes.md]: Extraneous "to" at the end of the sentence.

**Comment by stephamon** [comment]
👍

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a56e476f69/how-to/)

**Pull request merged**